### PR TITLE
fix(valid-lang): run on aria-hidden text

### PR DESCRIPTION
**Pull request by WilcoFiers**
Closes issue: #3633

### CODE REVIEW

File: test/commons/dom/is-hidden-with-css.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-describe('dom.isHiddenWithCSS', function() {
+describe('dom.isHiddenWithCSS', function () {
   'use strict';
 
   var fixture = document.getElementById('fixture');
   var shadowSupported = axe.testUtils.shadowSupport.v1;
-  var isHiddenWithCSSFn = axe.commons.dom.isHiddenWithCSS;
+  var isHiddenWithCSS = axe.commons.dom.isHiddenWithCSS;
```
Comment (WilcoFiers) [inline test/commons/dom/is-hidden-with-css.js:6]: I have no idea why we did this, but I thought it should be changed.

File: lib/commons/aria/get-element-unallowed-roles.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import getImplicitRole from './implicit-role';
 import getRoleType from './get-role-type';
 import isAriaRoleAllowedOnElement from './is-aria-role-allowed-on-element';
 import { tokenList, isHtmlElement, getNodeFromTree } from '../../core/utils';
-import AbstractVirtuaNode from '../../core/base/virtual-node/abstract-virtual-node';
+import AbstractVirtualNode from '../../core/base/virtual-node/abstract-virtual-node';
```
Comment (WilcoFiers) [inline lib/commons/aria/get-element-unallowed-roles.js:6]: Unrelated, but I spotted this typo. Figured I might as well fix it.

File: test/integration/rules/valid-lang/valid-lang.html [hunk 3]
```diff
@@ -10,11 +10,17 @@
 <p xml:lang="en-US" lang="gibberish-US" id="fail3">Mix</p>
 <p lang=" " id="fail4">English</p>
 
-<p lang="invalid" id="pass7">  <!-- ignore -->  </p>
+<p lang="invalid" id="pass7"><!-- ignore --></p>
```
Comment (WilcoFiers) [inline test/integration/rules/valid-lang/valid-lang.html:13]: Hmm, wonder why prettier did that.... Not sure it matters?

File: locales/_template.json [hunk 4]
```diff
@@ -965,7 +965,8 @@
         "default": "Element's default semantics were not overridden with role=\"none\" or role=\"presentation\"",
         "globalAria": "Element's role is not presentational because it has a global ARIA attribute",
         "focusable": "Element's role is not presentational because it is focusable",
-        "both": "Element's role is not presentational because it has a global ARIA attribute and is focusable"
+        "both": "Element's role is not presentational because it has a global ARIA attribute and is focusable",
+        "iframe": "Using the \"title\" attribute on an ${data.nodeName} element with a presentational role behaves inconsistently between screen readers"
```
Comment (straker) [inline locales/_template.json:969]: This has been fixed in #3632 and doesn't need to be in the pr.

File: lib/commons/dom/is-hidden-with-css.js [hunk 5]
```diff
@@ -10,8 +11,10 @@ import { getNodeFromTree } from '../../core/utils';
  * @param {Boolean} descendentVisibilityValue (Optional) immediate descendant visibility value used for recursive computation
  * @return {Boolean} the element's hidden status
  */
-function isHiddenWithCSS(el, descendentVisibilityValue) {
-  const vNode = getNodeFromTree(el);
+function isHiddenWithCSS(node, descendentVisibilityValue) {
```
Comment (straker) [inline lib/commons/dom/is-hidden-with-css.js:14]: This change isn't necessary once the isVisible refactor is in (the function name changed and it already handles virtual nodes). Do we still want to make this change if `isHiddenWithCSS` is being deprecated and not used anymore in our code?

**Review by straker — CHANGES_REQUESTED** [review]

File: lib/commons/dom/is-hidden-with-css.js (hunk 5 above)
Comment (WilcoFiers) [reply lib/commons/dom/is-hidden-with-css.js:14]: I didn't want to create a dependency on your refactor. I don't think doing both is a problem.

File: locales/_template.json (hunk 4 above)
Comment (WilcoFiers) [reply locales/_template.json]: Doesn't matter -- but I updated to develop.

**Review by straker — APPROVED** [review]

**Pull request merged**